app/index.js

```diff
@@ -40,21 +40,49 @@ if (!gotTheLock) {
 			notifications.addDesktopNotificationHack(iconPath);
 		}
 
-		window.webContents.on('new-window', (event, url, frame, disposition) => {
+		window.webContents.on('new-window', (event, url, frame, disposition, options) => {
+			console.debug('DEBUG - new-window triggered')
 			if (url.startsWith('https://teams.microsoft.com/l/meetup-join')) {
 				event.preventDefault();
 				window.loadURL(url);
 			} else if (disposition !== 'background-tab') {
 				event.preventDefault();
-				shell.openExternal(url);
+				if ((url === 'about:blank')) {
+					// Create a new hidden window to load the request in the background
+					console.debug('DEBUG - captured about:blank');
+					const win = new BrowserWindow({
+						webContents: options.webContents, // use existing webContents if provided
+						show: false
+					})
+					options.webContents.session.webRequest.onBeforeRequest((details, callback) => {
```

Comment:
While this won't be added until the first `new-window` request is made, it does mean we are now examining EVERY request going forward so this could impact the speed of the application or possibly cause some other issues. I didn't notice an issue in the few minutes I was running the app.

Reply:
I don’t think that will be a problem. I will test once I am back online! Might need to wait a few days... sorry!